src/components/Login.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default class Login extends React.Component {
+    render(){
+        return(
+            <LoginWrapperStyled> 
+                <h3>Log in here</h3>
+                <div>
+                <input type="text" name="name" placeholder="Name" required />
```

Comment:
Pay attention to indentation as this can make it harder to read.

Reply:
fixed